code/entities/cooking/hotdog-roller/HotdogRoller.Hotdog.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Sandbox;
+
+namespace Cinema.HotdogRoller;
+
+public partial class HotdogCookable : ModelEntity
+{
+    public bool Reversed { get; set; }
+    private Roller rollerParent {get; set;}
```

Comment:
Please follow style guide for naming variables. This should be `RollerParent`
https://github.com/tech-nawar/sbox-cinema/blob/main/STYLE.md#variable-naming-conventions